mobile-browser-based-version/src/task_definition/lus_covid.js

```diff
@@ -280,7 +280,7 @@ export const trainingInformation = {
     modelTrainData: {
         epochs: 10,
     },
-    threshold: 1,
+    threshold: 2,
```

Comment:
is this change intentional?

Reply:
Well, it depends on how many peers we want to wait for, so I increased in for the benchmarks. However, I suppose that in the future we can either adapt it dynamically or let the user set it in the interface. I'll still set it back to 1.